### PR TITLE
Added release notes for v0.15.0

### PR DESCRIPTION
**Pull request by writinwaters**
### Type of change

- [x] Documentation Update

### CODE REVIEW

**Comment by writinwaters** [comment]
You can merge it on the release day.

File: docs/release_notes.md [hunk 1]
```diff
@@ -7,6 +7,40 @@ slug: /release_notes
...
+Released on December 18, 2024.
+
+### New features
+
+- Introduces additional Agent-specific APIs.
+- Supports using page rank score to improve retrieval performance when searching across multiple knowledge bases.
+- Offers an iframe in Chat and Agent to facilitate the integration of RAGFlow into your webpage.
+- Adds a Helm chart for deploying RAGFlow on Kubernetes.
+- Supports importing or exporting an agent in JSON format.
+- Supports stepping for Agent components/tools.
```
Comment (JinHai-CN) [inline docs/release_notes.md]: stepping->step run

File: docs/release_notes.md [hunk 2]
```diff
@@ -7,6 +7,40 @@ slug: /release_notes
...
+- Adds a new UI language (*contributed by the community*): Japanese.
```
Comment (JinHai-CN) [inline docs/release_notes.md]: (*contributed by the community*) removed

File: docs/release_notes.md [hunk 3]
```diff
@@ -7,6 +7,40 @@ slug: /release_notes
 
 Key features, improvements and bug fixes in the latest releases.
 
+## v0.15.0
+
+Released on December 18, 2024.
+
+### New features
+
+- Introduces additional Agent-specific APIs.
```
Comment (JinHai-CN) [inline docs/release_notes.md:16]: Introduce a new layout recognition model in Deepdoc

**Pull request merged**